cfme/tests/services/test_rest_services.py

```diff
@@ -486,6 +487,8 @@ def test_delete_orchestration_templates(self, rest_api, orchestration_templates,
         collection = rest_api.collections.orchestration_templates
         if from_detail:
             methods = ['post', 'delete']
+            if BZ('1414881', forced_streams=['5.7', 'upstream']).blocks:
+                methods[0] = 'delete'  # because the 'post' method fails
```

Comment:
I can do that however reviewers wish, but if I add TODO, there's big chance it will get noticed long time after the bug is resolved. The way it is now the test will start working correctly right after the bug is resolved.
Also I wanted to leave the original code intact so that when it's time to clean up old workarounds and BZs, it's easy just to delete two lines without any other changes in the test necessary.